code/mod_table/mod_table.cpp

```diff
@@ -765,6 +766,10 @@ void parse_mod_table(const char *filename)
 
 		}
 
+		if (optional_string("$SCPUI attempts to load hires animations:")) {
```

Comment:
It's conceivable mods that don't use SCPUI may want to use this option, so I'd use `$Attempt to load hi-res animations:` or something similar.

Reply:
SCPUI refers to all the UI stuff available through librocket and this setting exclusively alters the way librocket libraries load animations for streaming.

Reply:
I was thinking about all the other uses of `generic_anim_init_and_stream`, not just the one in RocketRenderingInterface.cpp.  And all the other places `2_` is used.

However, point taken, that's unnecessarily generalizing a very specific feature.